web3/_utils/rpc_abi.py

```diff
@@ -51,6 +51,8 @@
     'eth_sendRawTransaction': ['bytes'],
     'eth_sendTransaction': TRANSACTION_PARAMS_ABIS,
     'eth_sign': ['address', 'bytes'],
+    'eth_submitHashrate': ['uint', 'bytes'],
```

Comment:
I think this line should be [bytes32, bytes32]?

Reply:
While parity's fine with that, it seems like geth doesn't like that, going with `['uint', 'bytes32']`

Reply:
Iiiinteresting. [geth's docs](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_submithashrate) are wrong then. Sounds good!